README.md

```diff
@@ -67,9 +85,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- This project is the Microverse's third module's capstone project
```

Comment:
- [ ] Kindly add proper credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) author of [the original design,](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) as required by the [Creative Commons](https://creativecommons.org/licenses/).